joomla/joomla-cms
# [5.2] [Guided Tours] Arrows do not show properly, especially in dark mode

### Summary of Changes

When introducing dark mode, a border was added to the tour popups so that the tours would be more visible in dark mode.
Unfortunately, this 'cuts' the arrows off the popup and no border was added to the arrows themselves so that they are not visible in dark mode.

This is the first attempt in fixing that.

Although this PR fixes the arrows, there is more to be done, particularly that the colors for the tour popups are too dark and barely pop over the content. Check PR#43854.

### Testing Instructions

Run the welcome tour in light and dark mode and check that the arrows are showing properly.

### Actual result BEFORE applying this Pull Request

A popup in light mode.
The popup border shows over the arrow and the arrow is not of the color of the header background.

![light_before](https://github.com/user-attachments/assets/72bf82b1-a3e0-45a5-aeb3-1d60fc6c46ce)

A popup in dark mode.
The border cuts off the arrow from the popup and is barely visible.

![dark_before](https://github.com/user-attachments/assets/d243ce58-677d-4f06-84a8-ac385fee9d94)

### Expected result AFTER applying this Pull Request

A popup in light mode.
The border is gone and the arrow is of the right color.

![light_after](https://github.com/user-attachments/assets/f8825448-86ba-4a7f-b0ca-49831c2661d1)

A popup in dark mode.
The border no longer cuts off the arrow from the popup and the arrow is more visible.

![image](https://github.com/user-attachments/assets/cd601ef9-0433-4fb6-be5c-8ac1281d03e7)

![image](https://github.com/user-attachments/assets/2c6257f3-6ebf-4842-94a1-565d47179c66)

### Link to documentations
Please select:
